entries/f/flaticon.com.json

```diff
@@ -3,7 +3,8 @@
     "domain": "flaticon.com",
     "contact": {
       "twitter": "flaticon",
-      "facebook": "flaticon"
+      "facebook": "flaticon",
```

Comment:
```suggestion
```

The [Facebook test is saying](https://app.circleci.com/pipelines/github/2factorauth/twofactorauth/8433/workflows/0cd3abf9-772f-4937-a62c-ee3ecfbb3289/jobs/8459?invite=true#step-106-4) that it's private or doesn't exist, even though it does.

Reply:
It's almost worth getting rid of the Facebook page, the test keeps failing regardless of if you put it's username or the page number